dev-preview/README.md

```diff
@@ -187,4 +79,27 @@ Follow the installation instructions in the above to install MCO operator, and e
 
 ### Usage
 
-Simply set `InstanceSize` field on MCO CR to a value that would suit your monitoring needs.
\ No newline at end of file
+Simply set `InstanceSize` field on MCO CR to a value that would suit your monitoring needs.
+
+
+# Graduated features
+
+### Hosted Control Planes with MCE (MCE 2.5)
+  
+  * Usage and use-case documentation can be found in the [Hosted Control Plane Clusters section of the doc](https://github.com/stolostron/hypershift-deployment-controller/blob/main/docs/content.md).  
+
+### Multicluster Global Hub (ACM 2.9)
+  
+  * Usage instructions and examples can be found in the [Getting Started Section](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#hosted-control-planes-intro).
```

Comment:
this should be a link to Global Hub: https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/multicluster_global_hub/index